runtime/pom.xml

```diff
@@ -229,6 +229,10 @@
               <goal>verify</goal>
             </goals>
             <configuration>
+              <additionalClasspathElements>
+                <!-- workaround suggested in https://github.com/spring-projects/spring-boot/issues/6254 -->
+                <additionalClasspathElement>${project.build.outputDirectory}</additionalClasspathElement>
```

Comment:
Nasty, a workaround, see the [issue](https://github.com/spring-projects/spring-boot/issues/6254)